packages/shorebird_cli/lib/src/auth/auth.dart

```diff
@@ -1,54 +1,104 @@
 import 'dart:convert';
 import 'dart:io';
 
+import 'package:googleapis_auth/auth_io.dart';
+import 'package:http/http.dart' as http;
 import 'package:path/path.dart' as p;
-import 'package:shorebird_cli/src/auth/session.dart';
 import 'package:shorebird_cli/src/config/config.dart';
 
+export 'package:googleapis_auth/googleapis_auth.dart' show AccessCredentials;
+
+final _clientId = ClientId(
+  /// Shorebird CLI's OAuth 2.0 identifier.
+  '30552215580-6kgm623s9l4te0rd63r18c5u6b6gkts5.apps.googleusercontent.com',
+
+  /// Shorebird CLI's OAuth 2.0 secret.
+  ///
+  /// This isn't actually meant to be kept secret.
+  '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',
```

Comment:
I'm confused.  Is this not supposed to be secret? https://www.oauth.com/oauth2-servers/client-registration/client-id-secret/

Reply:
Looks like the "secret" is optional? https://pub.dev/documentation/googleapis_auth/latest/googleapis_auth/ClientId-class.html

Reply:
https://docs.gitguardian.com/secrets-detection/detectors/specifics/google_oauth2_keys

Reply:
> The Client ID must be created with a client_secret here, however there is no way to properly secure a client_secret for installed/console applications. Fortunately the OAuth2 flow used in this case [assumes that the app cannot keep secrets](https://developers.google.com/identity/protocols/oauth2/native-app) so this particular client_secret does not need to be kept secret. You should however make sure not to re-use the same client_secret anywhere secrecy is required.

https://pub.dev/packages/googleapis_auth

Reply:
OK.  Do we need to leave ourselves more breadcrumbs here about why this isn't secret or what we can/can't do with it?  Thanks.